mlflow/store/artifact/databricks_sdk_models_artifact_repo.py

```diff
@@ -0,0 +1,93 @@
+import io
+import os
+import posixpath
+
+from mlflow.entities import FileInfo
+from mlflow.store.artifact.cloud_artifact_repo import CloudArtifactRepository
+
+def _get_databricks_workspace_client():
+    from databricks.sdk import WorkspaceClient
+
+    return WorkspaceClient()
+
+
+class DatabricksSDKModelsArtifactRepository(CloudArtifactRepository):
+    """
+    Stores and retrieves model artifacts via Databricks SDK, agnostic to the underlying cloud
+    that stores the model artifacts.
+    """
+
+    def __init__(self, model_name, model_version):
+        self.model_name = model_name
+        self.model_version = model_version
+        from mlflow.utils.logging_utils import eprint
+        eprint("Using DatabricksSDKModelsArtifactRepository")
+        model_name_parts = model_name.split(".")
+        catalog = model_name_parts[0]
+        schema = model_name_parts[1]
+        model = model_name_parts[2]
+        self.model_base_path = f"/Models/{catalog}/{schema}/{model}/{model_version}"
```

Comment:
```suggestion
        self.model_base_path = f"/Models/{catalog}/{schema}/{model}/{model_version}"
        self.client = _get_databricks_workspace_client()
```

can we instantiate the client here and reuse it?

Reply:
Is catalog/schema used anywhere else? If not can we use `"/Models/{model_name.replace(".", /)}/{model_version}"`